src/styles/blocks/card.scss

```diff
@@ -0,0 +1,91 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  @extend %card-border;
+  font-family: "Roboto", Helvetica, sans-serif;
+  font-style: normal;
+  margin: 0 48px 46px 0;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 40px;
+  }
+
+  &__title,
+  &__rating,
+  &__price,
+  &__pay {
+    margin: #{$global-margin};
+  }
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &__title {
+    &-name {
+      margin: 0;
+      @extend %product-name-text;
+      color: #{$main-color};
+    }
+
+    &-code {
+      margin: 4px 0 0;
+      @extend %product-code-text;
+      color: #{$secondary-color};
+      margin-bottom: 8px;
+    }
```

Comment:
It's not a modifiers or elements of `price`, it's a product-card block elements.
 More detail in video.
[video, timing 4:15](https://mate.academy/learn/css-advanced/bem#/video/911)
```suggestion
    &__title-name {
      margin: 0;
      @extend %product-name-text;
      color: #{$main-color};
    }

    &__title-code {
      margin: 4px 0 0;
      @extend %product-code-text;
      color: #{$secondary-color};
      margin-bottom: 8px;
    }
```
![Screenshot 2022-07-21 at 19 59 48](https://user-images.githubusercontent.com/17098736/180271524-a7f67f2a-8beb-4155-a497-95617757c49e.png)